ush/config.sh.RRFS_CONUS_13km

```diff
@@ -65,8 +66,12 @@ TILE_SETS="full NE,NC,NW,SE,SC,SW"
 DO_DACYCLE="TRUE"
 DO_SURFACE_CYCLE="TRUE"
 DO_SPINUP="TRUE"
+#DO_POST_SPINUP="TRUE"
+DO_POST_PROD="TRUE"
 DO_RETRO="TRUE"
 DO_NONVAR_CLDANAL="TRUE"
+#USE_RRFSE_ENS="TRUE"
+CYCL_HRS_HYB_FV3LAM_ENS=("01" "02" "03" "04" "05" "06" "07" "08" "09" "10" "11" "12" "13" "14" "15" "16" "17" "18" "19" "20" "21" "22" "23" "00")
```

Comment:
Suggest removing 03 and 15 from the list, since we can't use RRFS ensemble for the cold IC.

Reply:
03/15 has warm start GSI analysis for product cycles.